build/gen_stub.php

```diff
@@ -17,6 +17,10 @@
 error_reporting(E_ALL);
 ini_set("precision", "-1");
 
+if (PHP_VERSION_ID < 70400) {
+    throw new Exception("The build system requires PHP 7.4 at least.\n");
+}
```

Comment:
This check isn't going to work in that a parse error will be thrown before it runs.

Reply:
uh, yes... Then we can rely on the check in php.m4 only, I guess?

Reply:
D'oh! We *could* split the file (basically rename gen_stub.php to gen_stub.inc, and then have gen_stub.php check the PHP version, and include gen_stub.inc on success), but that might be overkill.

Reply:
Yeah, this idea came to my mind too, but neither I thought that it makes much sense.